modules/transactions/src/integrationTest/java/org/apache/ignite/internal/disaster/DisasterRecoveryTestUtil.java

```diff
@@ -125,27 +126,34 @@ static void assertValueOnSpecificNodes(
             int id,
             int val,
             SchemaDescriptor schema
-    ) throws Exception {
-        for (IgniteImpl node : nodes) {
-            assertValueOnSpecificNode(tableName, node, id, val, schema);
-        }
+    ) {
+        CompletableFuture<?>[] futures = nodes.stream()
```

Comment:
May the follow be simpler? Assertion method is synchronous, but `runAsync` still will use common fork-join-pool.

```
nodes.parallelStream().forEach(node -> assertValueOnSpecificNode(tableName, node, id, val, schema));
```

Reply:
changed to the original code

Reply:
Fine as for me